packages/radix-ui-themes/src/components/context-menu.tsx

```diff
@@ -197,17 +206,24 @@ ContextMenuRadioItem.displayName = 'ContextMenuRadioItem';
 type ContextMenuCheckboxItemElement = React.ElementRef<typeof ContextMenuPrimitive.CheckboxItem>;
 type ContextMenuCheckboxItemOwnProps = GetPropDefTypes<typeof contextMenuCheckboxItemPropDefs>;
 interface ContextMenuCheckboxItemProps
-  extends React.ComponentPropsWithoutRef<typeof ContextMenuPrimitive.CheckboxItem>,
+  extends PropsWithoutRefOrColor<typeof ContextMenuPrimitive.CheckboxItem>,
     ContextMenuCheckboxItemOwnProps {}
 const ContextMenuCheckboxItem = React.forwardRef<
   ContextMenuCheckboxItemElement,
   ContextMenuCheckboxItemProps
 >((props, forwardedRef) => {
-  const { children, className, shortcut, ...itemProps } = props;
+  const {
+    children,
+    className,
+    shortcut,
+    color = contextMenuCheckboxItemPropDefs.color.default,
```

Comment:
We should use the new extract function for those here right @vladmoroz?

Reply:
I remember I hadn't used it on the item because effectively it wasn't doing anything useful for us there anyway.

Let's settle this separately—maybe there's an argument for using it on all props everywhere just for consistency, but there are other components/parts like that.